src/twisted/conch/ssh/transport.py

```diff
@@ -279,6 +281,24 @@ def verify(self, seqid, data, mac):
         outer = hmac.HMAC(self.inMAC.key, data, self.inMAC[0]).digest()
         return hmac.compare_digest(mac, outer)
 
+    def isOutMACETM(self) -> bool:
+        """
+        @return True if outgoing MAC is ETM.
+        """
+        return self.outMACType.endswith(b"-etm@openssh.com")
```

Comment:
Feedback from @glyph 

Get this value cached.

We can pre-compute them when we set the key.